app/frontend/src/components/agriseafoodopscreening/admin/AdminReviewSubmission.vue

```diff
@@ -41,21 +41,21 @@
 <script>
 import { mapGetters } from 'vuex';
 
-import Step2 from '@/components/minesoperatorscreening/Step2.vue';
-import Step3 from '@/components/minesoperatorscreening/Step3.vue';
-import Step4 from '@/components/minesoperatorscreening/Step4.vue';
-import Step5 from '@/components/minesoperatorscreening/Step5.vue';
+import Step1 from '@/components/agriseafoodopscreening/Step1.vue';
+import Step2 from '@/components/agriseafoodopscreening/Step2.vue';
+import Step3 from '@/components/agriseafoodopscreening/Step3.vue';
+import Step4 from '@/components/agriseafoodopscreening/Step4.vue';
 
 export default {
-  name: 'ReviewSubmission',
+  name: 'AdminReviewSubmission',
```

Comment:
Probably needs an agriculture-specific name for now or would be a duplicate?

Reply:
We can get by for now having a generic name as the DOM by how we organize won't have collisions - we only have one type of this component per view. But yes, either we refactor long-term so this is actually only one common component, or we make the name more specific based on directory context too.

Let's revisit this when we have a chance to refactor...